lib/ansible/plugins/connection/napalm.py

```diff
@@ -112,6 +112,8 @@
         key: connect_timeout
     env:
       - name: ANSIBLE_PERSISTENT_CONNECT_TIMEOUT
+    vars:
+      - name: ansible_connect_timeout
```

Comment:
Since the connection is initiated even before task variable is read this option might not work for per task basis